types/oauth/index.d.ts

```diff
@@ -0,0 +1,316 @@
+// Type definitions for oauth 0.9
+// Project: https://github.com/ciaranj/node-oauth#readme
+// Definitions by: nonAlgebraic <https://github.com/nonAlgebraic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.7
```

Comment:
why use `2.7`? this means that this declaration file can not be used by any one using a TS version less than 2.7. i do not see any 2.7-specific features in the file.. so is this intentional?